src/DependencyInjection/Configuration.php

```diff
@@ -30,6 +30,15 @@ public function getConfigTreeBuilder(): TreeBuilder
             ->addDefaultsIfNotSet()
             ->children()
                 ->scalarNode('driver')->defaultValue(SyliusResourceBundle::DRIVER_DOCTRINE_ORM)->end()
+                ->arrayNode('generation')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('exclude_root_taxon')
+                            ->info('Exclude root taxon while generating product type?')
+                            ->defaultFalse()
+                        ->end()
+                    ->end()
+                ->end()
```

Comment:
Remove this again. This is very related to the specific feed type, i.e. the Google feed. In the future we should have such configuration when creating the feed

Reply:
Ok